### DISCUSSION TITLE
Random app icon

### DISCUSSION THREAD

**Discussion by Milo123459** [community/community#3574] [Mobile]
A setting allowing for a random app icon to be picked from the available ones in settings and just changes it at 00:00 each day

**Comment by PranamBhat** [comment]
Hi @Milo123459 

Kindly elaborate your issue. Thank you.

**Reply by Milo123459** [reply]
The section in settings that allows you to change the app icon, it'd be cool if everyday it picked a random one from that list.

**Comment by osamashadwo** [comment]
Send to me plez
